content/blog/2020/08/2020-08-25-external-fingerprint-storage-phase-3.adoc

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
+title: "External Fingerprint Storage Phase-3 Update: Introducing the PostgreSQL Fingerprint Storage Plugin"
```

Comment:
```suggestion
title: "Introducing the PostgreSQL Fingerprint Storage Plugin"
```

Maybe? A lot of duplication otherwise. Please feel free to reject, and I will publish it as is then.